### PR TITLE
Add Docker registry option to Drone builds

### PR DESCRIPTION
**Pull request by Menda**
Missing feature to have Docker registry configuration.

### CODE REVIEW

File: service_builder/setup_service.py [hunk 1]
```diff
@@ -193,15 +203,25 @@ def setup():
...
-    if is_answer_yes:
+    is_answer_yes_docker = yes_or_no('Add Docker support?')
+    if is_answer_yes_docker:
         _configure_docker(name_project)
 
-    is_answer_yes = yes_or_no(
-        'Add Drone CI test support?')
-    if is_answer_yes:
+    is_answer_yes_drone = yes_or_no('Add Drone CI test support?')
+    if is_answer_yes_drone:
```
Comment (bogdangi) [inline service_builder/setup_service.py]: this condition is not tested.

File: service_builder/setup_service.py [hunk 2]
```diff
@@ -193,15 +203,25 @@ def setup():
...
-    is_answer_yes = yes_or_no('Add Docker support?')
-    if is_answer_yes:
+    is_answer_yes_docker = yes_or_no('Add Docker support?')
+    if is_answer_yes_docker:
         _configure_docker(name_project)
 
-    is_answer_yes = yes_or_no(
-        'Add Drone CI test support?')
-    if is_answer_yes:
+    is_answer_yes_drone = yes_or_no('Add Drone CI test support?')
```
Comment (bogdangi) [inline service_builder/setup_service.py]: Why just do it like
```python 
if yes_or_no('Add Drone CI test support?'):
    _configure_drone_ci()
```

File: service_builder/setup_service.py [hunk 3]
```diff
@@ -182,6 +182,16 @@ def _configure_drone_ci():
                          'web administration panel')
 
 
+def _configure_docker_registry(name_project: str, registry_domain: str,
+                               registry_folder: str):
+    registry_url = os.path.join(registry_domain, registry_folder, name_project)
+    filename = os.path.join('drone-ci', '.drone-appendix.yml')
+    content = get_template_content(filename)
+    content = content.replace('{{ registry_domain }}', registry_domain)
```
Comment (bogdangi) [inline service_builder/setup_service.py]: Why do not use templating engine to do not invent homegrown solution.

File: service_builder/setup_service.py (hunk 2 above)
Comment (Menda) [reply service_builder/setup_service.py]: Because we need this variable for later

File: service_builder/setup_service.py (hunk 1 above)
Comment (Menda) [reply service_builder/setup_service.py]: Done

File: service_builder/setup_service.py (hunk 3 above)
Comment (Menda) [reply service_builder/setup_service.py]: You are right. It can be done as an improvement for a later PR.

**Pull request merged**